src/components/CampaignOverview.vue

```diff
@@ -0,0 +1,167 @@
+<template>
+  <section class="favor-ranking">
+    <table class="favor-ranking-table tableFixHead">
+      <thead class="sticky">
+        <tr>
+          <th class="favor-ranking-rank">Rank</th>
+          <th>Faction</th>
+          <th class="favor-ranking-seasons">Spring</th>
+          <th class="favor-ranking-seasons">Summer</th>
+          <th class="favor-ranking-seasons">Fall</th>
+          <th class="favor-ranking-seasons">Winter</th>
+          <th class="favor-ranking-favor">Favor</th>
+        </tr>
+        <tr>
+          <td colspan="2" class="favor-ranking-DFR">Divine Favor Ranking</td>
+          <td class="favor-ranking-spring"></td>
+          <td class="favor-ranking-summer"></td>
+          <td class="favor-ranking-fall"></td>
+          <td class="favor-ranking-winter"></td>
+          <td></td>
+      </tr>
+      </thead>
+      <tbody>
+        <tr v-for="(elem, index) in data" v-bind:key="elem.faction">
+          <td>{{index + 1}}</td>
+          <td class="favor-ranking-faction-elem"><span class="favor-ranking-faction-red"></span>{{elem.faction}}</td>
+          <td>
+            <table class="inner-table">
+              <tr>
+                <td v-for="num in elem.spring" v-bind:key="elem.faction+num">{{num}}</td>
+              </tr>
+            </table>
+          </td>
+          <td>
+            <table class="inner-table">
+              <tr>
+                <td v-for="num in elem.summer" v-bind:key="elem.faction+num">{{num}}</td>
+              </tr>
+            </table>
+          </td>
+          <td>
+            <table class="inner-table">
+              <tr>
+                <td v-for="num in elem.fall" v-bind:key="elem.faction+num">{{num}}</td>
+              </tr>
+            </table>
+          </td>
+          <td>
+            <table class="inner-table">
+              <tr>
+                <td v-for="num in elem.winter" v-bind:key="elem.faction+num">{{num}}</td>
+              </tr>
+            </table>
+          </td>
+          <td>{{ [...elem.spring, ...elem.summer, ...elem.fall, ...elem.winter].reduce((a, b) => a + b, 0) }}</td>
```

Comment:
just as a good practice: better to not have 1-letter variables, `a` might be named as a `sum`, and `b` something like `points`